### DISCUSSION TITLE
Multi-world linking?

### DISCUSSION THREAD

**Discussion by Ghoose1** [Pryaxis/TShock#2223] [General]
is there a way to have a server that can switch players between two worlds without them having to disconnect / reconnect, and allow different people to be on both servers at the same time?

**Comment by dark22** [comment]
Currently it is possible through [Dimensions](https://github.com/popstarfreas/Dimensions) built with public source code by Popstarfreas, owner of the Dark-Gaming server. If this is useful, I recommend that if it is not uncomfortable for you, that you make a donation of any amount to the maintainer.
